secp256k1-sys/README.md

```diff
@@ -1,21 +1,25 @@
-secp256k1-sys
-=============
+# Secp256k1-sys (FFI for libsecp256k1)
 
+[![Build Status](https://travis-ci.org/rust-bitcoin/rust-secp256k1.png?branch=master)](https://travis-ci.org/rust-bitcoin/rust-secp256k1)
 
-This crate provides Rust definitions for the FFI structures and methods.
+[Full documentation](https://docs.rs/secp256k1_sys/)
+
+This is the Foreign Function Interface (FFI) to the [libsecp256k1](https://github.com/bitcoin-core/secp256k1) library.
+Provides bindings (structures and methods) in Rust for interfacing with `libsecp256k1` in a typesafe manner.
```

Comment:
The `-sys` crate is not type safe.

Reply:
We could drop the "in a typesafe manner" clause, but in fairness, the bindings *are* typesafe in that they enforce that you use the correct C types, correct number of arguments to functions, etc.

Reply:
> you use the correct C types

That's questionable at best. If you happen to be on a platform that defines `c_int` as `i32` and you accidentally hard-code `i32` in the code your code suddenly becomes non-portable to platforms that have `c_int` be a different type (presumably `i16` or `i64`.

They are "sort of safe" yes.